ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.9.yaml

```diff
@@ -122,8 +127,10 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
-    - chain: openshift-upgrade-qe-test-chainupgrade
```

Comment:
Why we need this change? I saw we are reference `openshift-upgrade-qe-test-chainupgrade` even in 4.8 -> 4.17 chain upgrade.

Reply:
I refereed https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.5.yaml#L112 which is the latest aws-ipi job.

Reply:
The source version is 4.5, that is too old, I guess that is because olm does not support yet on 4.5, so we have to split the install workflow.

For the source version on 4.9, I think we should use install workflow and upgrade chain as usual.

Reply:
ok, let us try the new workflow

Reply:
done